tests/conftest.py

```diff
@@ -22,23 +22,52 @@ def mimir_es(docker_services):
 def mimir_client(mimir_es):
     return Elasticsearch([mimir_es])
 
+
 @pytest.fixture(scope='session')
 def wiki_es(docker_services):
     """Ensure that ES is up and responsive."""
     docker_services.start('wiki_es')
     port = docker_services.wait_for_service("wiki_es", 9200)
 
-    url = f'http://{docker_services.docker_ip}:{port}'
+    temp_wiki_es = settings._settings['WIKI_ES']
+    temp_es_wiki_lang = settings._settings['ES_WIKI_LANG']
 
+    url = f'http://{docker_services.docker_ip}:{port}'
     settings._settings['WIKI_ES'] = url
     settings._settings['ES_WIKI_LANG'] = 'fr'
+
     return url
 
+    settings._settings['WIKI_ES'] = temp_wiki_es
+    settings._settings['ES_WIKI_LANG'] = temp_es_wiki_lang
 
 @pytest.fixture(scope='session')
 def wiki_client(wiki_es):
     return Elasticsearch([wiki_es])
 
+
+@pytest.fixture(scope='session')
+def wiki_es_ko(docker_services):
+    docker_services.start('wiki_es')
+    port = docker_services.wait_for_service("wiki_es", 9200)
+
+    temp_wiki_es = settings._settings['WIKI_ES']
```

Comment:
since it's something that seems to be done often, I think it's worth it do make it more generic.

maybe something like (not tested, just speudo code):

```python
@pytest.fixture(scope='session')
def wiki_es_ko(docker_services):
    ...
    with utils.override_settings({'WIKI_ES': url, 'ES_WIKI_LANG': 'fr'}):
        yield
```

and in some test/utils.py file
```python
from contextlib import contextmanager
from copy import deepcopy
from app import settings

@contextmanager
def override_settings(overrides):
    old_settings = deepcopy(settings._settings)
    settings._settings.update(overrides)
    yield 
    settings._settings = old_settings
 ```

Reply:
Nice,
done